Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.h

```diff
@@ -61,12 +62,22 @@ class AnnotationTrackingState {
     bool m_isBeingHovered { false };
 };
 
-enum class WebEventModifier : uint8_t;
-
 enum class AnnotationSearchDirection : bool {
     Forward,
     Backward
 };
+
+struct PerPageInfo {
+    PDFDocumentLayout::PageIndex pageIndex { 0 };
```

Comment:
isn't this somewhat duplicative of PDFDocumentLayout's bounds array?

Reply:
It is, but I can't access that off the main thread.